core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -665,15 +665,6 @@ fold_bind_results([#kz_responder{module=M
         {'error', _E}=E ->
             lager:debug("error: ~p", [_E]),
             E;
-        {'EXIT', {'undef', [{_M, _F, _A, _}|_]}} ->
-            ST = erlang:get_stacktrace(),
-            log_undefined(M, F, length(Payload), ST),
```

Comment:
From [this](https://gist.github.com/fenollp/a362f1dfa94f2033700ba20d092c8588) it looks like the stacktrace is `[]` (between `try` and `catch`). So `log_undefined` would never log anything.

Still want me to do this? Would be dead code to me.